problem_builder/public/js/step.js

```diff
@@ -4,6 +4,8 @@ function MentoringStepBlock(runtime, element) {
 
     var submitXHR, resultsXHR,
         message = $(element).find('.sb-step-message');
+    
+    var childManager = new ProblemBuilderUtilLMS.ChildManager(element, runtime);
```

Comment:
@jbzdak Now that the utils file has been renamed, `ProblemBuilderUtilLMS` should be renamed as well I think. How about just `StepUtil`?